docs/hugo/content/contributing/create-a-new-release.md

```diff
@@ -22,6 +22,23 @@
    ```
 9. Make sure they deploy successfully - check in the portal as well.
 
+# Creating and testing Helm chart for new release
+1. Create a new branch from `<NEW_RELEASE_TAG>` HEAD
+2. Generate helm manifest for new release: `task controller:gen-helm-manifest`
+3. Check the version in `/v2/charts/azure-service-operator/Chart.yaml` if matches with the latest release tag.
+4. Install helm chart:
+```
```

Comment:
Minor: I think this renders better in a list in markdown if you indent it 4 spaces. That way it looks like the codeblock is in list item 4.

Reply:
Done!